TheGladiator/Assets/Scripts/Manager/MasterManager.cs

```diff
@@ -28,8 +30,8 @@ private void Awake()
         managerList.Add(ManagerLoadScene);
         managerList.Add(ManagerGlobalData);
         managerList.Add(ManagerSound);
+        managerList.Add(ManagerSprite);
```

Comment:
Need to assign ManagerSprite using getComponent<SpriteManager> before adding it into the managerlist
See the code between line 24 and line 29